Library/Formula/sstp-client.rb

```diff
@@ -0,0 +1,33 @@
+class SstpClient < Formula
+  homepage "http://sstp-client.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/sstp-client/sstp-client/1.0.9/sstp-client-1.0.9.tar.gz"
+  sha1 "b56c60fc6ecd8b1686d6c319a8adb3f988bcabbd"
+
+  depends_on "libevent"
+  depends_on "openssl"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--disable-ppp-plugin",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+
+    # Create a directory needed by sstpc for privilege separation
+    mkdir_p "var/run/sstpc"
+  end
+
+  def caveats; <<-EOS.undent
+    sstpc reads PPP configuration options from /etc/ppp/options. If this file
+    does not exist yet, type the following command to create it:
+
+    sudo touch /etc/ppp/options
```

Comment:
I have read the source code of `pppd` and it looks like `/etc/ppp/options` is required to exist *only* if `pppd` is *not* run as root; invoking `pppd` as root or via `sudo` seems to work even if `/etc/ppp/options` does not exist. So, an alternative could be to ask the user to make `sstpc` setuid root so it can execute `pppd` as root. I'm not sure that it's safe to do so, but other formulae (like `htop`) also propose the user to make certain executables setuid root, so maybe that's not a big deal (and `sstpc` drops root privileges anyway when it does not need root access any more).